<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1597</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>-10</y>
      <width>901</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pet Image Segmentation with Python Deep Learning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGoruntuSec">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>750</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Görüntü Seç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGoruntuKaydet">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>750</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Görüntüyü Kaydet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>760</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Giriş Görüntüsü</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>760</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Çıkış Görüntüsü</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGirisGoruntusu">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCikisGoruntusu">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1597</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
